main.rb

```diff
@@ -4,6 +4,7 @@
 require 'mongoid'
 require 'nokogiri'
 require 'json'
+require 'slack/post'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.